docker/supervisord.conf

```diff
@@ -19,14 +19,24 @@ port = 127.0.0.1:9001
 command=redis-server
 priority=1
 
-[program:celery]
-command=celery -A memex worker -l info --loglevel=debug --logfile=memex/logs/celeryd.log
-priority=2
-
 [program:elasticsearch]
 command=elasticsearch
 priority=1
 
+[program:nutch]
+command=nutch startserver
+priority=1
+
+[program:rabbitmq]
+command=sudo rabbitmq-server ; need administrative access on Linux systems
```

Comment:
Needs sudo on Linux, I've noted this in https://github.com/memex-explorer/memex-explorer/issues/725

cc @brittainhard